resources/js/components/SearchForm.vue

```diff
@@ -0,0 +1,161 @@
+        viewHandler(e) {
+            if (e.percentInView === 1 || e.percentTop > .2 && e.percentTop < .8) {
+                document.querySelector(`#${e.target.element.id[0]}-search-button`).classList.add('bolt');
+            }
+            else {
+                document.querySelector(`#${e.target.element.id[0]}-search-button`).classList.remove('bolt');
+            }
+        }
+    },
+    filters: {
+        momentDate(date) {
+            return moment(date).date();
+        },
+        momentWeek(date) {
+            return moment(date).format('MMMM');
+        }
+    },
+};
+</script>
+
+<template>
+    <div>
+        <div class="ris-committee__wrapper">
+            <div class="ris-search ris-committee__search">
+                <button class="ris-search__button">
+                    <span class="ris-i ris-i_search"/>
+                </button>
+                <input type="search" class="ris-search__input" v-model="inputValue"
+                    @input="filterList"
+                    placeholder="Suche nach Themen, Vorlagen, Sitzungen..."
+                        >
+            </div>
+            <div class="ris-select ris-committee__select">
+                <div class="ris-select__label">Darstellung</div>
+
+                <select class="ris-select__select">
+                    <option class="ris-select__option" data-sort-type="newest-first">
+                        Das Neuste zuerst
+                    </option>
+                    <option class="ris-select__option" data-sort-type="oldest-first">
+                        Chronologische Reihenfolge
+                    </option>
+                </select>
+                <span class="ris-i ris-i_chevron-double"/>
+            </div>
+        </div>
+        <nav v-if="!filtered" class="ris-committee-navigation">
+            <ul class="ris-ul ris-committee-navigation__list">
+                <li
+                    v-for="(item, index) in sortedCommittees"
+                    class="ris-committee-navigation__item"
+                    :key="`${index}-list-button`">
+                    <button
+                        @click="scrollToComponents(item.char)"
+                        :id="`${item.char}-search-button`"
+                        class="ris-committee-navigation__button">
+                        {{ item.char }}
+                    </button>
+                </li>
+            </ul>
+        </nav>
+        <transition-group tag="ul" name="fade" class="ris-committee-main-list ris-ul" v-if="!filtered">
+            <li v-for="item in sortedCommittees"
+                class="ris-committee-main-list__item"
+                :id="`${item.char}-list-element`"
+                v-view="viewHandler"
+                :key="item.char">
+                <h2 class="ris-committee-main-list__heading ris-h2">{{ item.char }}</h2>
+                <ul class="ris-ul ris-committee-secondary-list">
+                    <li v-for="(committe, committeIndex) in item.data"
```

Comment:
i recommend to make a single component out of this and https://github.com/nextsolx/ratsinformationssystem/blob/5a9614c21694b3f94c058c5edd144255a5224bb1/resources/js/components/SearchForm.vue#L148
then look the same for me